src/store/user/index.js

```diff
@@ -236,10 +220,18 @@ const actions = {
         }
     },
 
-    async getUserProfile({ commit, state }) {
+    async updateUserInfo({ commit, state }) {
         try {
+            await mayaClient.post('/auth/update-fields', state.userProfile);
+        } catch (err) {
+            throw new Error(err);
```

Comment:
if we need to throw then why are we catching at first place.

Reply:
added to do comment.. will write later the exception case